index.html

```diff
@@ -0,0 +1,254 @@
+<!-- QUIZ TIME
```

Comment:
Never put a comment before the doctype. The browser needs to know it's dealing with HTML before anything else.